lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -1051,38 +1064,40 @@ mod tests {
         assert_eq!(ClipboardPDUType::CB_FORMAT_LIST, header.msg_type);
         assert_eq!(1, format_list.format_names.len());
         assert_eq!(
-            ClipboardFormat::CF_OEMTEXT as u32,
+            ClipboardFormat::CF_TEXT as u32,
             format_list.format_names[0].format_id
         );
 
         // verify that the clipboard data is now cached
         // (with a null-terminating character)
         assert_eq!(
             String::from("abc\0").into_bytes(),
-            *c.clipboard
-                .get(&(ClipboardFormat::CF_OEMTEXT as u32))
-                .unwrap()
+            *c.clipboard.get(&(ClipboardFormat::CF_TEXT as u32)).unwrap()
         );
     }
 
     #[test]
     fn update_clipboard_conversion() {
-        for (input, expected) in &[
-            ("abc\0", "abc\0"),       // already null-terminated, no conversion necessary
-            ("\n123", "\r\n123\0"),   // starts with LF
-            ("def\r\n", "def\r\n\0"), // already CRLF, no conversion necessary
-            ("gh\r\nij\nk", "gh\r\nij\r\nk\0"), // mixture of both
+        struct Item(&'static [u8], &'static [u8], ClipboardFormat);
+        for Item(input, expected, format) in [
+            Item(b"abc\0", b"abc\0", ClipboardFormat::CF_TEXT), // already null-terminated, no conversion necessary
+            Item(b"\n123", b"\r\n123\0", ClipboardFormat::CF_TEXT), // starts with LF
+            Item(b"def\r\n", b"def\r\n\0", ClipboardFormat::CF_TEXT), // already CRLF, no conversion necessary
+            Item(b"gh\r\nij\nk", b"gh\r\nij\r\nk\0", ClipboardFormat::CF_TEXT), // mixture of both
+            Item(
```

Comment:
Would be good to add a test that is UTF-8 (not ASCII) and also needs CRLF conversion.

Reply:
Addressed